docs-site/src/examples/allow_invalid_dates.jsx

```diff
@@ -21,10 +21,10 @@ export default React.createClass({
     return <div className="row">
       <pre className="column example__code">
         <code className="jsx">
-          {"<DatePicker"}<br />
-              <strong>{"allowInvalidDates"}<br /></strong>
-              {"selected={this.state.startDate}"}<br />
-              {"onChange={this.handleChange} />"}
+          {'<DatePicker'}<br />
```

Comment:
These are changes to satisfy eslint; they were a result of the merge of the old and new forks.